hw02/PlayerController.cs

```diff
@@ -14,18 +25,51 @@ public class PlayerController : MonoBehaviour
                 1. Player or Character's Name
                 2. Movement Speed as a float
                 3. Gravity Value as a float
-                4. Jump Speed as a float
-    
+                4. Jump Speed as a float 
+    */
+
+    // Answer 2.1
+    // These are the variables that we will be using to control our player.
+    private string playerName = "Player";
+    private float movementSpeed = 10.0f;
+    private float gravityValue = 9.8f;
+    private float jumpSpeed = 10.0f;
+    // If we want change the name of player through the console, we can use the following code:
+    playername=Console.ReadLine();
+    Console.WriteLine("The player's name is: " + playerName);
+    // If we want to print the values of the variables to the Unity, we can use the following code:
+    Debug.Log("The player's name is: " + playerName);
+    Debug.Log("The player's movement speed is: " + movementSpeed);
+    debug.Log("The player's gravity value is: " + gravityValue);
+    Debug.Log("The player's jump speed is: " + jumpSpeed);
+    // These lines of code will print the values of the variables to the Unity console
+    // while we need to put them in the Start() or suitable function.    
+
+
+    /*
     When the game starts, we would like to find our character. In our Unity Editor, we have a 3D model of our player and the player is 
     represented as a component in Unity. How do we grab our player controller in code and where should we write this line of code in this
     C# document (PlayerController.cs)?
     */
-    
+
+    // Answer 2.2
+    private GameObject player;
+    // We can grab our player controller in code by using the following line of code:
+    GameObject player = GameObject.Find("Player");
```

Comment:
If we are using RigidBody we want to do something similar to: 
`myRigidbody = gameObject.GetComponent<Rigidbody>();`.